multicluster/apis/multicluster/v1alpha1/clusterset_types.go

```diff
@@ -27,14 +27,23 @@ type MemberCluster struct {
 	ClusterID string `json:"clusterID,omitempty"`
 	// API server of the destination cluster.
```

Comment:
"API server endpoint of the cluster. Used only for a leader cluster".

Maybe you can also give an example string here?

Reply:
Updated.